.github/workflows/pythonbuild.yml

```diff
@@ -255,6 +255,12 @@ jobs:
           - python-version: 3.11
             plugin-names: "flytekit-whylogs"
     steps:
+      # Clean up unused tools to have more disk space in the GitHub hosted runner.
+      # https://github.com/seal-io/walrus/pull/2111
+      - name: Free disk space
```

Comment:
the `integration` job uses https://github.com/flyteorg/flytekit/blob/master/.github/workflows/pythonbuild.yml#L141 to reclaim disk space. That takes care of removing those directories and more. IMO we should adopt that in the `build` and `build-plugins` jobs.

Reply:
updated it